### PR TITLE
EN-336 Upgrade data object mapper management

### PR DESCRIPTION
**Pull request by eugeniosant**
Upgrade of Cache Info Manager to facilitate insert of Infinispan connector (into components project).

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/aps/system/services/dataobjectmapper/cache/DataObjectMapperCacheWrapper.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import org.entando.entando.aps.system.services.dataobjectmapper.DataObjectPageMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author E.Santoboni
+ */
+public class DataObjectMapperCacheWrapper extends AbstractCacheWrapper implements IDataObjectMapperCacheWrapper {
+
+	private static final Logger _logger = LoggerFactory.getLogger(DataObjectMapperCacheWrapper.class);
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/dataobjectmapper/cache/DataObjectMapperCacheWrapper.java]: Remove underscore.

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Pull request merged**